supported-version/src/versions/magento-open-source/individual.json

```diff
@@ -669,6 +697,20 @@
         "nginx": "nginx:1.22",
         "os": "ubuntu-latest",
         "release": "2025-02-11T00:00:00+0000",
+        "eol": "2025-04-07T00:00:00+0000"
+    },
+    "magento/project-community-edition:2.4.6-p10": {
+        "magento": "magento/project-community-edition:2.4.6-p10",
+        "php": 8.1,
+        "composer": "2.2.21",
+        "mysql": "mysql:8.0",
+        "elasticsearch": "elasticsearch:8.5.3",
```

Comment:
```suggestion
        "elasticsearch": "elasticsearch:8.17.4",
```

Reply:
@rhoerr 2.4.6-p10 doesn't support elastic so we should probably remove this line?